iptvtools/models.py

```diff
@@ -144,20 +144,23 @@ def filter(self):
         random.shuffle(urls)
         pbar = tqdm(urls, ascii=True)
         for url in pbar:
-            time.sleep(self.args.interval)
             status = 'OK'
-            if self.args.min_height or self.args.resolution_on_title:
-                height = utils.check_stream(url, self.args.timeout)
-                if height == 0:
+            if not self.args.no_connectivity_check:
+                time.sleep(self.args.interval)
```

Comment:
Is this necessary? I suppose you want a quick check when skip connectivity check. I think this can be done by setting the interval? `-I/--interval`?

Reply:
yes but wats the point to wait the interval if no connectivity check should be done? thats why is in the if